src/ui/tabs/main_tab.py

```
@@ -38,6 +38,7 @@ def __init__(self, main_window_ref):
         self.additional_amount_value_label = None
         self.in_total_value_label = None
         self.main_calculate_button = None
+        self.save_to_cloud_button = None # New button for saving to cloud
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)